src/msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,170 @@
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the web API.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|---------------------------------:|:------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `active-directory-dotnet-minimal-api-aspnetcore-client-credentail-flow`      | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**                 | _None_                                                                       | No redirect URI required; don't select a platform.                                                                    |
+| **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Configure the web API
+
+1. Open the _~/msal-client-credentials-flow/appsettings.json_ file in your code editor and modify the following values values with those from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis):
+
+   ```json
+   "ClientId": "Enter_the_Application_Id_here",
+   "TenantId": "Enter_the_Tenant_Info_Here",
+   "ClientSecret": "Enter_the_Application_CLient_Secret_Here",
+   ...
+   "RelativePath": "Enter_the_Application_Object_Id_Here",
+   ...
+   ```
+
+## Run the application
+
+### 1. Run the web API
+
+1. Execute the following command to get the app up and running:
+
+   ```bash
+   dotnet run
+   ```
+
+### 2. Send request to the web API
+
+1. Once the app is listening, execute the following to send the a request.
+
+   ```bash
+   curl -X GET https://localhost:5001/application -ki
+   ```
+
+   :information_source: Since the request is sent without a Bearer Token, it is expected to receive an Unauthorized response `401`. The web API is now protected
```

Comment:
That's not in scope for this scenario, the endpoint can just be anon.

Reply:
good catch indeed @ckittel dragged it from another scenario I made before.

done | addressed from [ea1ed](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/22/commits/ea1ed059c9d6236c9939802815224b454416bb86)

Reply:
🪓